scripts/preinstall.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
+
+var lsp_server_version = "1.0.0-SNAPSHOT";
+
+const download = require("mvn-artifact-download").default;
 const fs = require('fs');
-const download = require('download')
-var url = "https://github.com/camel-tooling/camel-language-server/releases/download/untagged-27364bc5f4ff78ca37a9/camel-lsp-server-1.0.0-SNAPSHOT.jar"
+const path = require('path');
 
-download(url).then(data => {
-    fs.writeFileSync('./jars/language-server.jar', data);
+download('com.github.camel-tooling:camel-lsp-server:' + lsp_server_version, './jars/', 'https://oss.sonatype.org/content/repositories/snapshots/').then((filename)=>{
```

Comment:
is it possible to not specify a version? at least when there is an error? (can be done in a separated PR, in this case we can open another jira issue)

Reply:
not necessary due to same reasons than here https://github.com/camel-tooling/camel-lsp-client-vscode/pull/28#discussion_r198081467